src/mds/topology/topology_chunk_allocator.cpp

```diff
@@ -175,9 +175,12 @@ bool TopologyChunkAllocatorImpl::ChooseSingleLogicalPool(
         if (!topology_->GetPhysicalPool(lPool.GetPhysicalPoolId(), &pPool)) {
             continue;
         }
-        uint64_t diskCapacity = pPool.GetDiskCapacity();
-        // calculate actual capacity available
-        diskCapacity = diskCapacity * poolUsagePercentLimit_ / 100;
```

Comment:
poolUsagePercentLimit_ is required, indicating the percentage of chunkfilepool usage